explorer/assets/css/app.css

```diff
@@ -50,13 +50,45 @@
 }
 
 @layer components {
-	.tooltip {
-		display: none;
-	}
+    .tooltip {
+        display: none;
+    }
+
+    .tooltip[data-show] {
+        position: absolute;
+        display: block;
+        z-index: 9999;
+    }
 
-	.tooltip[data-show] {
-		position: absolute;
-		display: block;
-		z-index: 9999;
-	}
+    /* Progress bar based on https://dev.to/mikekennedydev/progress-bar-text-colour-1gg6
+     * Foreground color removed to use the accent colors */
+    .progress-with-labels {
```

Comment:
We are using [tailwind](https://tailwindcss.com/docs/utility-first), maybe we can style it in the component itself via classes instead of css?